### ISSUE TITLE
IDEATION: When clicking on an idea in closed ideation, takes me to voting tab

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#1329]
Can't open the idea, just get redirected to the voting tab. Only seems to happen for one particular idea. Other ideas I can open fine. Was an idea that wasn't chosen to be included in the voting, may be connected?

**Comment by ilmartyrk** [comment]
@BeccaMelhuish please add more context, is it a public or a private topic, were you logged in or logged out, if member then with what permissions

**Comment by BeccaMelhuish** [comment]
I'm relatively sure it was a public topic for which I was admin and I was logged in, and the idea was one that wasn't 'chosen' to be included in the vote. (Potentially was one that had been reported, was a fake spam one). But I'm struggling to track it down right now..

**Comment by BeccaMelhuish** [comment]
@ilmartyrk a similar thing has happened, so I can give more details:

In [this public topic](https://test.app.citizenos.com/topics/join/gljY9sbyoyQf), for which I am admin.  

When clicking 'View anyway' on an idea in a closed idea gathering that's been deleted, instead of viewing the idea, I get taken to the 'Action' (Follow-up) tab: 
![image](https://github.com/citizenos/citizenos-fe/assets/61320526/9f2d7c9c-7d43-42cc-a5e0-e3d063765d12)

**Comment by BeccaMelhuish** [comment]
I've noticed in the recording that the user in this topic had a similar but slightly different problem with the same idea. When she clicked 'view anyway', she just got taken to the top of the same page (it didn't open, basically just refreshed the page). 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/ab6f17c6-cd9c-46d1-85ed-b4701715d833)

View after clicking: 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/22cce82d-7583-4f89-a1af-a7f50277781b)

**Comment by BeccaMelhuish** [comment]
Moved back to testing as think that's where it's meant to be?

**Comment by ssin1901** [comment]
@BeccaMelhuish in the Test environment I sent a topic to voting and add 2/3 ideas. Then I went back to the idea gathering and clicked on the one that was not selected. All worked fine. I checked the others and all seem fine. This was the view (screenshot1) and I was not taken to the voting tab.

Can we send to Done?

<img width="313" alt="Image" src="https://github.com/user-attachments/assets/7cadac0d-280e-4ee3-8a6b-4665cc21763e" />
